File: Mobile/polydessinmobile/app/src/main/java/com/example/polydessinmobile/Lobby.kt
package com.example.polydessinmobile

data class Lobby(val gameId: String = "",
                 val mode: String = "",
                 val difficulty: String = "",
                 val players: Map<String, Int> = mapOf("player1" to 0),
                 val host: String = "",
                 val name: String = "",
                 @field:JvmField
                 val isPublic: Boolean? = null,
                 val shareKey: String = ""
)

File: Remise/Serveur/firebase-cloud-functions/functions/src/interfaces/lobby.ts
import {GameDifficulty} from "../enums/game-difficulty";
import {GameModes} from "../enums/game-modes";
import {PlayerPositions} from "../enums/player-positions";

export interface Lobby {
    id: string,
    name: string,
    gameId: string,
    difficulty: GameDifficulty,
    host: string,
    mode: GameModes,
    players: Map<string, PlayerPositions>,
    isPublic: boolean,
    shareKey: string
}
